doc/man1/flux-mini.rst

```diff
@@ -57,6 +57,16 @@ over time, making them suitable for use in scripts.
 
 The available OPTIONS are detailed below.
 
+.. note::
+
+  These commands target the *enclosing instance*. For example, within
+  a *SCRIPT* submitted with ``flux mini batch``, a ``flux mini`` command
+  will submit a job to the batch instance, not the parent instance under
+  which the batch instance is running. To target the parent instance,
+  the ``--parent`` option of :man1:`flux` should be used, e.g::
+
+   $ flux --parent mini batch [OPTIONS].. SCRIPT
+
```

Comment:
I think your example is meant to illustrate what to put in a script, not a command line?  Perhaps it'd be more clear if you did:

```
#!/bin/sh
flux --parent mini batch ...
```

Reply:
Good point. I'll just drop the prompt since putting the shebang may make some readers think the command goes first in the script.